platforms/js/build_js.py

```diff
@@ -256,7 +260,8 @@ def build_loader(self):
                         help="Specify configuration file with own list of exported into JS functions")
     parser.add_argument('--webnn', action="store_true", help="Enable WebNN Backend")
 
-    args = parser.parse_args()
+    transformed_args = ["--cmake_option=%s".format(arg) if arg[:2] == "-D" else arg for arg in sys.argv[1:]]
```

Comment:
`.format` can't handle `%s` because it uses `{}`

Reply:
Sorry I didn't catch this back then, I usually use the `f''` format and didn't realise that `.format` needed different placeholders. During my retest it probably just ended up using a default value, which was correct for my setup.

Would you like me to do a quick followup to modify the format string?